<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIInitialFieldDialog</class>
 <widget class="QWidget" name="GUIInitialFieldDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Nameï¼š</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditName">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelType">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditType">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelKeyWord">
     <property name="text">
      <string>KeyWord:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditKeyWord">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelSet">
     <property name="text">
      <string>Set:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="GUI::NodeGroupWidget" name="widgetSet" native="true"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelCoordinateSystemID">
     <property name="text">
      <string>Coordinate System:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditCoordinateSystem"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Initial Value</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Vx:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditVX"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Vy:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditVY"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Vz:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditVZ"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUI::NodeGroupWidget</class>
   <extends>QWidget</extends>
   <header>GUIDialog/GUIAnalysisDialog/NodeGroupWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
